csharp/sources/Valkey.Glide.InterOp/Native/CommandResult.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.ComponentModel;
+using System.Diagnostics.CodeAnalysis;
+using System.Runtime.InteropServices;
+
+namespace Valkey.Glide.InterOp.Native;
+
+[SuppressMessage("ReSharper", "InconsistentNaming")]
+[StructLayout(LayoutKind.Sequential, CharSet = CharSet.Unicode)]
+[EditorBrowsable(EditorBrowsableState.Advanced)]
+public struct CommandResult
+{
+    public        int   success;
+    public unsafe byte* error_string;
```

Comment:
Use `IntPtr` to avoid unsafe

Reply:
These structs are inherently unsafe and not intended for end users. I can do it, but it won’t fully convey just how unsafe this structure actually is.